README.md

```diff
@@ -98,6 +98,10 @@ text_font=sans-serif
 - [Arch Linux (git)](https://aur.archlinux.org/packages/swappy-git)
 - [Fedora 31/32](https://copr.fedorainfracloud.org/coprs/wef/swappy)
 - [openSUSE](https://build.opensuse.org/package/show/X11:Wayland/swappy)
+- Void Linux
+  ```bash
+  sudo xbps-install swappy
+  ```
```

Comment:
Can you simply point to the package like it's done above ? It's up the to distro users to know how to install those packages.
Also the commit is not respecting `conventional-changelog`.

Reply:
The [online package catalog for Void Linux](https://voidlinux.org/packages/) doesn't include the package name in the URL.
Also, Void Linux doesn't have much package managers, like Arch Linux, just CLI XBPS, and a GUI for it (that nobody uses).
Would it be better to have this?
> Void Linux: the `swappy` package in the repositories.

Reply:
Sure sounds good enough